AKS-Hybrid/deploy-gpu-node-pool.md

```diff
@@ -82,7 +107,8 @@ When you uninstall the host driver, the physical GPU goes into an error state. Y
 For each GPU (3D Video Controller) device, run the following commands in PowerShell. Copy the instance ID; for example, `PCI\VEN_10DE&DEV_1EB8&SUBSYS_12A210DE&REV_A1\4&32EEF88F&0&0000` from the previous command output:
 
 ```powershell
-$id1 = "<Copy and paste GPU instance id into this string>"
+$gpu=Get-PnpDevice -FriendlyName "3D Video Controller"
+$id1 =$gpu[0].InstanceId
```

Comment:
These works for single GPU device. But if there're multiple GPU installed on one host, we need a loop. Moreover, end users may want to choose to setup some, but not all of devices.

Reply:
This is called out on line 107 with the statement "For Each GPU...., run the following commands in Powershell."  Do we need to make it even more clear?

Reply:
In line 110 the `$gpu` gets all devices named as "3D Video Controller", so in line 111 the `$id1` will always be set to the instance id of the first GPU device.